src/common/core/core.js

```diff
@@ -1659,6 +1663,173 @@ define([
          * @func
          */
         this.canSetAsMixin = core.canSetAsMixin;
+
+        //library function TODO checking everything and adding all new functions
+
+        /**
+         * It adds a project as library to your project by copying it over. The library will be a node
+         * with the given name directly under your project's ROOT. It becomes a read-only portion of your project.
+         * You will only be able to manipulate it with library functions, but cannot edit the individual nodes inside.
+         * However you will be able to instantiate or copy the nodes into other places of your project. Every node
+         * that was part of the META in the originating project becomes part of your project's meta.
+         * @param {module:Core~Node} node - any regular node in your project.
```

Comment:
This is good doc (some tweaks can be made) I would however extract the conceptual things to a wiki-page. And assume the core user know what a library is.